src/factory.js

```diff
@@ -68,6 +69,8 @@ export class Factory {
     }
     this.renderQ = [];
     this.stave = null; // current stave
+
+    this.StaveNote = this.StaveNote.bind(this);
```

Comment:
Can you elaborate on why this does not belong here? This seems appropriate if the point of the Factory is to provide a common "construction context" for objects, unless there's a specific reason to allow usage of these methods as unbounded functions -- but I can't think of one, since that's what plain constructors are for. I suppose it's important to note that I didn't add this specifically for the tests, but to minimize boilerplate/duplication around usage of the Factory _in general_. (And I think this should be extended to every factory method)

Binding instance methods on construction is not an uncommon pattern in the JS ecosystem, for example, `class` based React components often employ this pattern.